### ISSUE TITLE
SAC in statistics not calculated for dives with >1 gas and incomplete info

### ISSUE THREAD

**Issue by sfuchs79** [subsurface/subsurface#1269]
### Describe the issue:

- [ ] Bug
- [ ] Change request
- [x] New feature request
- [ ] Discussion request

### Issue long description:

Dive with 3 different gases. No pressure info from DC. Manually entered start and end pressure for 2 out of 3 gases.
Display of SAC rate ("AMV" - sorry German) in dive info tab is correct:
![grafik](https://user-images.githubusercontent.com/25316050/39971179-5c22795c-56f7-11e8-86b4-8da0b43565c4.png)

Display of SAC in statistics tab is not available.

### Operating system:

Windows 10

### Subsurface version:

Current master 4.7.8-something

### Steps to reproduce:

Example dive:
[SAC_Example.zip](https://github.com/Subsurface-divelog/subsurface/files/1998869/SAC_Example.zip)

**Comment by atdotde** [comment]
What is the expected behaviour? Not computing a SAC at all as information is missing?

**Comment by atdotde** [comment]
Do you have gas switch events to these gases? In that case, we might be able to compute per gas SACs.

**Comment by sfuchs79** [comment]
Yes, gas switch events are there. Per gas SAC is already displayed correctly in dive info tab:
![grafik](https://user-images.githubusercontent.com/25316050/40021329-c77af7ba-57c4-11e8-9c85-af4f33526007.png)

In the statistics tab I would expect to see an correct average SAC based on the sections of the dive for which a per gas SAC is available. This would mean for the example dive a value between 14,0 and 16,8l/min.

**Comment by sfuchs79** [comment]
This was addressed here by disabling the incorrect SAC calculation:
https://github.com/Subsurface-divelog/subsurface/pull/1289/commits/6f42a33d0ee7eed52ca2418366aeac125d289c76

I would like to keep the issue open for the moment because maybe someone finds the time to implement a CORRECT SAC calculation for the above described use case :-)
